docs/rackhd/samples/dhcpd-non-rackhd.conf

```diff
@@ -0,0 +1,17 @@
+ddns-update-style none;
+option domain-name "example.org";
+option domain-name-servers ns1.example.org, ns2.example.org;
+
+default-lease-time 600;
+max-lease-time 7200;
+log-facility local7;
+
+deny duplicates;
+
+ignore-client-uids true;
+
+subnet 172.31.128.0 netmask 255.255.240.0 {
+ range 172.31.128.3 172.31.143.254;
+ # Use this option to signal to the PXE client that we are doing proxy DHCP
+ option vendor-class-identifier "PXEClient";
```

Comment:
Remember the case when installing Ubuntu (net installer), is it suitable to add below to this example ?
```
option routers 172.31.128.1;
option domain-name-servers 10.254.174.10;
```